recipes-wpe/wpeframework/wpeframework_git.bb

```diff
@@ -90,7 +97,8 @@ def getlayerrevision(d):
 
 WPE_LAYER_REV ??= "${@getlayerrevision(d)}"
 
-EXTRA_OECMAKE += " -DINSTALL_HEADERS_TO_TARGET=ON \
+EXTRA_OECMAKE += "\
+    -DINSTALL_HEADERS_TO_TARGET=ON \
```

Comment:
I'm a bit worried about that flag... Is it something legacy?

Reply:
Removed this, since no longer required